pkg/controller/workloadspread/workloadspread_controller_utils.go

```diff
@@ -78,3 +80,14 @@ func filterOutCondition(conditions []appsv1alpha1.WorkloadSpreadSubsetCondition,
 	}
 	return newConditions
 }
+
+func nodeMatchesSubset(node *corev1.Node, subset *appsv1alpha1.WorkloadSpreadSubset) (bool, error) {
```

Comment:
Here, should we first merge the affinity and tolerations of pod with subset, and then match the node? @FillZpp @furykerry